po/oc.po

```diff
@@ -399,28 +392,20 @@ msgid "_Browse"
 msgstr "Percórrer"
 
 #: src/exm-window.blp:102
-#, fuzzy
-#| msgid "Sort Enabled First"
 msgid "Sort _Enabled First"
-msgstr "Las activadas en primièr"
+msgstr "Las _activadas en primièr"
```

Comment:
This string and the followings have access keys (they are triggered with <kbd>Alt</kbd>+letter, <kbd>a</kbd> in this case) in the same menu, so the underline character has to be in front of different letters for each one.